src/dao.ts

```diff
@@ -49,7 +49,7 @@ export class DAO implements IStateful<IDAOState> {
         nativeReputation { id, totalSupply }
         nativeToken { id, name, symbol, totalSupply }
         membersCount
-        members (where: {address:"${this.address}"}) {
+        members {
```

Comment:
wait why? We are getting the totken and rep balance of the dao here.

Reply:
I thought it is done automatically. Maybe I'm wrong.